deploy/nft.ts

```diff
@@ -18,7 +22,33 @@ const deployNFT: DeployFunction = async (hre) => {
     hre,
   })
 
-  const nftDistributor = await deployDiamond<ITellerNFTDistributor>({
+  let execute: DeployDiamondArgs<ITellerNFTDistributor, any>['execute']
+  try {
+    // Try to get deployment of TellerDiamond
+    await contracts.get('TellerNFTDistributor')
```

Comment:
This is also dependent on the TellerDiamond yes?

Reply:
yea, if it throws, then we deploy a new one, calling the `initialize` function as the `execute` parameter